test/test_multiscale_spatial_image.py

```diff
@@ -66,6 +66,11 @@ def test_isotropic_scale_factors(input_images):
     image = input_images[dataset_name]
     baseline_name = "2_4/XARRAY_COARSEN"
     multiscale = to_multiscale(image, [2, 4], method=Methods.XARRAY_COARSEN)
+    store = DirectoryStore(
+        DATA_PATH / f"baseline/{dataset_name}/{baseline_name}", dimension_separator="/"
+    )
+    multiscale.to_zarr(store)
```

Comment:
My impression is that this will override the local baseline with the test output so that a potentially different baseline image is loaded the next time `pytest` is called. Is this intended?

Reply:
Yes -- we can explicitly set `mode = "w"` here, although this is the default.